src/updates.js

```diff
@@ -213,28 +234,31 @@ class Updates {
       PLATFORM_ARCH.WIN_IA32,
       PLATFORM_ARCH.WIN_ARM64,
     ]) {
-      if (latest[key]) {
-        const rurl = `https://github.com/${account}/${repository}/releases/download/${latest[key].version}/RELEASES`;
+      const the_latest = latest[key];
```

Comment:
This block contains the only functional changes I made. Their only purpose is to get better type support, since `latest[key]` shows up as `Latest|undefined` even inside the `if(latest[key]){}` block.